### DISCUSSION TITLE
Guys, add a radar!!!

### DISCUSSION THREAD

**Discussion by letovskiiy** [MartinTheDragon/Nuclear-Tech-Mod-Remake#76] [General]
Hello, dear developers, friends!
Please add the radar from the latest 1.7.10 release to this version of the game!!! Please, I beg you very much!
![image](https://github.com/MartinTheDragon/Nuclear-Tech-Mod-Remake/assets/160428452/757d6c6f-0b0b-4b05-b5b4-98faffb3e556)
![image](https://github.com/MartinTheDragon/Nuclear-Tech-Mod-Remake/assets/160428452/f0e5335c-6c71-4f9e-b05b-da3e13c5a9c6)
![image](https://github.com/MartinTheDragon/Nuclear-Tech-Mod-Remake/assets/160428452/efab4578-86e1-4b62-bc14-7868ed88cd50)

**Comment by MartinTheDragon** [comment]
Hi!
The radar will get added when the time is right, though you may have to wait for a good while. Someone already made an issue for the radar, so this is pretty much a duplicate of #63.
I'm sorry content like that hasn't been ported yet or isn't getting ported as quickly as could be, but development just has been a bit slow recently. Currently focused is Project Sorcerer, which is a compiler plugin effort to get the port to multiple versions, and I also migrated to [Codeberg.org](https://codeberg.org/MartinTheDragon/Nuclear-Tech-Mod-Remake).
I could've continued working on the port while also working on the compiler plugin, however I chose to focus on the plugin instead whenever I work on the port, so I'm sorry that your pleas won't change the situation.
See ya when the port picks up speed again!

**Reply by letovskiiy** [reply]
Thanks for such a quick response bro!!! Can you tell me if the HBM mod on Fabric is suitable for playing on the server version 1.18.2?

**Reply by MartinTheDragon** [reply]
I'm neither aware of a Fabric port, nor would I particularly recommend playing with the current state of the port on a survival server unless you do a bit of tweaking or add recipes for current content by yourself.
